packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -44,6 +44,18 @@ const {
 const { stripIndent } = require(`common-tags`)
 const { prefixId, CODES } = require(`./error-utils`)
 
+let warnedForDeprecation = false
+
+function warnForDeprecation() {
+  if (warnedForDeprecation) {
+    return
+  }
+  warnedForDeprecation = true
+  console.warn(
+    `[gatsby-transformer-sharp] The "fixed" and "fluid" resolvers are now deprecated. Switch to the new Gatsby image plugin for better performance and a simpler API. See https://gatsby.dev/migrate-images to learn how.`
```

Comment:
I'd say `gatsby-plugin-image` so they know what it's called even in this message.